.claude/skills/translate-batch/scripts/watch-pr.sh

```
@@ -0,0 +1,119 @@
+set -uo pipefail
+
+REPO="${1:?usage: watch-pr.sh <owner/repo> <pr_number>}"
+PR="${2:?usage: watch-pr.sh <owner/repo> <pr_number>}"
+POLL_INTERVAL="${POLL_INTERVAL:-60}"
+PENDING_TIMEOUT="${PENDING_TIMEOUT:-900}"
+
+now_iso() { date -u +%Y-%m-%dT%H:%M:%SZ; }
+emit() { printf '[%s] %s\n' "$(now_iso)" "$*"; }
+
+prev=""
+start_ts=$(date +%s)
```

Shellcheck (0.11.0)

[warning] start_ts appears unused. Verify use (or export if used externally).

(SC2034)